posthog/models/insight.py

```diff
@@ -30,6 +30,7 @@ class Insight(models.Model):
     description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
     filters: models.JSONField = models.JSONField(default=dict)
+    # stores the filters hash for this insight when displayed in isolation, not on a dashboard
```

Comment:
Adding this comment as `help_text=""` means it gets displayed in the API docs

Reply:
Oh, that's cool... I didn't realise you could put the help_text on models as well as serializer fields

We replace filters_hash when returning the response so I put the text on the serializer instead... which made me realise it wasn't read only and really should be

Reply:
(very quickly) added to docs https://github.com/PostHog/posthog.com/pull/3367